utils/loadArcherModules

```diff
@@ -0,0 +1,28 @@
+#%Module
+#
+# DCProgs
+#
+
+#module-whatis "Loads modules to compile and test DCProgs in ARCHER."
+
+# Use the right version of CMake - Note only CMake 2.6 is available in the compute nodes
+module unload cmake
+module load cmake/3.2.3
+
+# Swig is needed by the python bindings
+module load swig
+
+# Load Python3
+module load anaconda-compute/2.2.0-python3
+
+# Python tests are run with behave
+# Behave won't work unless we add its install location to $PATH
+pip install behave
+export PATH=~/.local/bin:$PATH
+
+# Use GNU compiler suite environment
+module unload PrgEnv-cray
+module load PrgEnv-gnu/5.2.56
```

Comment:
Do this before installing python modules to make sure any python modules are build with gcc